server/tests-py/test_graphql_queries.py

```diff
@@ -632,24 +632,19 @@ def dir(cls):
         return 'queries/explain'
 
     def test_simple_query(self, hge_ctx):
-        self.skip_if_pg_9_5(hge_ctx)
         self.with_admin_secret(hge_ctx, self.dir() + '/simple_query.yaml')
 
     def test_permissions_query(self, hge_ctx):
-        self.skip_if_pg_9_5(hge_ctx)
         self.with_admin_secret(hge_ctx, self.dir() + '/permissions_query.yaml')
 
     def with_admin_secret(self, hge_ctx, f):
         conf = get_conf_f(f)
         admin_secret = hge_ctx.hge_key
+        headers = {}
         if admin_secret:
-            conf['headers'] = {
-                'X-Hasura-Admin-Secret': admin_secret
-            }
-        check_query(hge_ctx, conf, add_auth=False)
-
-    def skip_if_pg_9_5(self, hge_ctx):
-        pg_version = hge_ctx.pg_version
-        # skip tests for PG version < 9.6 since the plan response varies in the same
-        if pg_version < 90600:
-            pytest.skip('Skip explain query tests for Postgres version < 9.6')
+            headers['X-Hasura-Admin-Secret'] = hge_ctx.hge_key
+        status_code, resp_json, _ = hge_ctx.anyq(conf['url'], conf['query'], headers)
```

Comment:
A comment would be good here, explaining why we are only testing the `sql` key and not the plan.